custom_components/tuya_local/devices/dynasty_btx_fireplace.yaml

```diff
@@ -0,0 +1,193 @@
+name: Fireplace
+products:
+  - id: 5081205010521cfac95f
```

Comment:
Is this the product id, or the device id? (it should be the former, which is alphanumeric, not normally limited to hex characters as device ids often are)

Reply:
Updated to the correct product ID and removed the range for the temperature